### PR TITLE
docs: add short tutorials

### PR DESCRIPTION
**Pull request by Tier4Guan**
## Description

Add a lane change tutorial and an avoidance one.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by satoshi-ota — COMMENTED** [review]
@Tier4Guan Could you fix PR title and CI error? We need to follow [comventional commit](https://www.conventionalcommits.org/en/v1.0.0/) not only PR commit but also title.

https://github.com/autowarefoundation/autoware-documentation/actions/runs/5053064750/jobs/9066541081?pr=377

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 1]
```diff
@@ -81,6 +81,44 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Lane change scenario
+
+1. Download and unpack Nishishinjuku map.
+
+```bash
+gdown -O ~/autoware_map/ 'https://github.com/tier4/AWSIM/releases/download/v1.1.0/nishishinjuku_autoware_map.zip'
```
[the remark this answers is not in the document]
Comment (Tier4Guan) [reply docs/tutorials/ad-hoc-simulation/planning-simulation.md:89]: @satoshi-ota Thank you for the comment! But I tried with these two commands, and the zip could be downloaded and unzipped normally.

[the remark this answers is not in the document]
Comment (satoshi-ota) [reply docs/tutorials/ad-hoc-simulation/planning-simulation.md:89]: Sorry! I confirmed that the map download by this command.

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 2]
```diff
@@ -81,6 +81,44 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
...
+```bash
+source ~/autoware/install/setup.bash
+ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/autoware_map/nishishinjuku_autoware_map vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
+```
+
+![open-nishishinjuku-map](images/planning/lane-change/open-nishishinjuku-map.png)
+
+3. Set an initial pose and a goal pose in adjacent lanes.
+
+   ![set-postition-and-goal](images/planning/lane-change/set-postition-and-goal.png)
```
Comment (satoshi-ota) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:104]: @Tier4Guan Please fix typo.

**Review by satoshi-ota — APPROVED** [review]
@Tier4Guan Good job :+1:

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 3]
```diff
@@ -81,6 +81,44 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Lane change scenario
+
+1. Download and unpack Nishishinjuku map.
+
+```bash
```
Comment (satoshi-ota) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:88]: I guess you have to fix indent.

**Pull request merged**